### PR TITLE
Lei Wang needs to post CodingBat Exercise2

### PR DESCRIPTION
**Pull request by ghost**
I have completed my Codingbat.com part 2 exercises. Please review the post here: http://leiw414.github.io/spring2014/2014/02/15/Lei_PythonBasics2.html

### CODE REVIEW

File: _posts/2014-02-15-Lei_PythonBasics2.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+```
+def lone_sum(a, b, c):
+  sum = 0
+  if a not in [b,c]:
+    sum += a 
+  if b not in [a,c]:
+    sum += b 
+  if c not in [a,b]:
+    sum += c 
+  return sum
```
Comment (eah13) [inline _posts/2014-02-15-Lei_PythonBasics2.md:47]: Very clever using `not in`.

File: _posts/2014-02-15-Lei_PythonBasics2.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+**3. I found the most hardest is list-2 exercise called sum67. A lot of cases should be taken into consideration. At first, I came up with a script as following.**
+
+```
+def sum67(nums):
+  for i in nums:
+    if 6 in nums:
+      six = nums.index(6)
+      seven = nums.index(7)
+      for i in range(six,seven+1):
+        nums.pop(six)
```
Comment (eah13) [inline _posts/2014-02-15-Lei_PythonBasics2.md:60]: Very nice!  Have you encountered `pop` methods before?  They're quite useful sometimes.

File: _posts/2014-02-05-testytest.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (eah13) [inline _posts/2014-02-05-testytest.md]: @leiw414 you'll have to delete or rename this file- it's the same name as an existing file.

File: _posts/2014-02-15-Lei_PythonBasics2.md (hunk 2 above)
Comment (ghost) [inline _posts/2014-02-15-Lei_PythonBasics2.md:60]: I have used PHP array_pop method before, I was wondering if there was a
similar method in Python. Thus, I searched and got the "pop" method. In
PHP, array_pop only pop the last element in the array, while python's pop
more flexible, we can choose which element to pop. Python is definitely an
amazing programming language!

File: _posts/2014-02-05-testytest.md (hunk 3 above)
Comment (ghost) [inline _posts/2014-02-05-testytest.md]: I have deleted that post, so I should make a new pull request, right?
Thanks,
Lei

File: _posts/2014-02-15-Lei_PythonBasics2.md (hunk 2 above)
Comment (eah13) [inline _posts/2014-02-15-Lei_PythonBasics2.md:60]: Glad you're enjoying it!  `pop` originally referred to programming with [stacks](http://en.wikipedia.org/wiki/Stack_%28abstract_data_type%29) - Python lets you pop out of the middle, which is convenient but can be more difficult for a computer - easy for us, though, which is the point!

**Comment by eah13** [comment]
:boom:

**Pull request merged**